pubspec.yaml

```diff
@@ -14,7 +14,7 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  image: ^3.0.2
```

Comment:
Does it require the newer `image` dependency? If not, you can just set `>=3.0.2 < 5.0.0` and allow people to use a wider range of image dependencies.

Reply:
Yes, version 4 is needed